http-client/src/main/java/org/triplea/http/client/FeignFactory.java

```diff
@@ -0,0 +1,49 @@
+package org.triplea.http.client;
+
+import java.net.URI;
+
+import com.google.common.base.Preconditions;
+
+import feign.Feign;
+import feign.Logger;
+import feign.Request;
+import feign.gson.GsonDecoder;
+import feign.gson.GsonEncoder;
+import lombok.extern.java.Log;
+
+@Log
+final class FeignFactory {
+  private static GsonEncoder gsonEncoder = new GsonEncoder();
+  private static GsonDecoder gsonDecoder = new GsonDecoder();
```

Comment:
I don't think those variables are never getting reassigned, so making them final would be good

Reply:
c0ce634e1ab236bd2aa5e35ef80f6d4ee3002456

While there I also weakened the variable types.